src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8549,7 +8549,7 @@ namespace UnitarySystems {
                         if (CoolingLoad) Par[6] = 1.0;
                         Par[7] = 1.0;               // FLAG, 0.0 if latent load, 1.0 if sensible load to be met
                         Par[8] = OnOffAirFlowRatio; // Ratio of compressor ON mass flow rate to AVERAGE mass flow rate over time step
-                        Par[9] = 0.0;               // HXUnitOn is always false for HX
+                        Par[9] = HXUnitOn;          // HXUnitOn turns on or off HX in coil system
```

Comment:
Not sure why the HXUnitOn flag was always set to 0 here. Unit tests show it needs to be controlled based on the flag.